targets/TARGET_Realtek/TARGET_AMEBA/RTWInterface.cpp

```diff
@@ -255,6 +255,7 @@ nsapi_error_t RTWInterface::disconnect()
         return NSAPI_ERROR_DEVICE_ERROR;
     }
     while(1) {
+        wait(0.1);
```

Comment:
This needs own commit, explaining why is this wait needed here - a fix.

Reply:
Ok, I'm going to make own commit with explaining.